src/index.html

```diff
@@ -4,24 +4,407 @@
+        <div class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+
+        <div class="card__code">Product code: 195434</div>
+        <div class="review">
+          <div class="stars">
+            <div class="stars__star-active"></div>
+            <div class="stars__star-active"></div>
+            <div class="stars__star-active"></div>
+            <div class="stars__star-active"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="review__reviews">Reviews: 5</div>
+        </div>
+        <div class="card__price">
+          <div class="card__price-key">Price:</div>
+          <div class="card__price-value">$2,199</div>
+        </div>
+        <a
+          href="#buy"
+          data-qa="card-hover"
+        >
+          <button
+            type="submit"
+            class="button"
+          >
+            BUY
+          </button>
+        </a>
+      </div>
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__image"></div>
+        <div class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+
+        <div class="card__code">Product code: 195434</div>
+        <div class="review">
+          <div class="stars">
+            <div class="stars__star-active"></div>
+            <div class="stars__star-active"></div>
+            <div class="stars__star-active"></div>
+            <div class="stars__star-active"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="review__reviews">Reviews: 5</div>
+        </div>
+        <div class="card__price">
+          <div class="card__price-key">Price:</div>
+          <div class="card__price-value">$2,199</div>
+        </div>
+        <a
+          href="#buy"
+          data-qa="card-hover"
+        >
+          <button
+            type="submit"
+            class="button"
+          >
+            BUY
+          </button>
+        </a>
+      </div>
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__image"></div>
+        <div class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+
+        <div class="card__code">Product code: 195434</div>
+        <div class="review">
+          <div class="stars">
+            <div class="stars__star-active"></div>
+            <div class="stars__star-active"></div>
+            <div class="stars__star-active"></div>
+            <div class="stars__star-active"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="review__reviews">Reviews: 5</div>
+        </div>
+        <div class="card__price">
+          <div class="card__price-key">Price:</div>
+          <div class="card__price-value">$2,199</div>
+        </div>
+        <a
+          href="#buy"
+          data-qa="card-hover"
+        >
+          <button
```

Comment:
The 'type' attribute for a button inside an anchor (<a>) tag should be 'button', not 'submit'. 'submit' is used when the button is inside a form and is used to submit the form.